app/index.jsx

```diff
@@ -7,7 +7,7 @@ import { Provider } from "react-redux"
 import thunk from "redux-thunk"
 import isDev from "electron-is-dev"
 
-import DevTools from "./devtools"
+import DevTools from "./DevTools"
```

Comment:
There's some places in here where we weren't matching the casing when importing. Doesn't affect runtime, but tooling can be more stringent about these.